### DISCUSSION TITLE
How to see output during script execution

### DISCUSSION THREAD

**Discussion by djalan** [dkron-io/dkron#1575] [Q&A] [answered]
Hello

I would like to see the output of my script in the Dkron UI as my script is running. I tried two different executors, *files* and *log*, but I only see the output after my script exits.

Usually, the magic ingredient in Python is to call the executable with the unbuffered switch `-u`, but even this is not working.

```powershell
C:/python.exe -u C:/script.py
```

Version: dkron_3.2.7_windows_amd64

Any thoughts?

**Comment by djalan** [comment, marked as answer]
If you look at a job and its Execution in the Jobs tabs, you will not see the output.

On the other hand, the output is partially visible in the Busy tab.

![image](https://github.com/user-attachments/assets/fec0a7da-9e51-449e-8f9f-e057b585adaa)

**Comment by vcastellm** [comment]
You should be able to view the streaming output in http://dkronhost:8080/ui/#/jobs/<job>/show/executions in the UI click on the job then the Executions tab:
<img width="321" alt="image" src="https://github.com/user-attachments/assets/0f72b94a-6671-4228-89bb-da9ad7089d2f">

You should use the "shell" executor and no processor, so no "log" or "files" processor.
